mmdet/core/post_processing/bbox_nms.py

```diff
@@ -46,6 +48,12 @@ def multiclass_nms(multi_bboxes,
     if bboxes.numel() == 0:
         bboxes = multi_bboxes.new_zeros((0, 5))
         labels = multi_bboxes.new_zeros((0, ), dtype=torch.long)
+
+        # onnx export
+        tracing_state = torch._C._get_tracing_state()
+        if tracing_state:
```

Comment:
`torch.onnx.is_in_onnx_export()` may be helpful.

Reply:
Maybe we have to still use the current method. is_in_onnx_export use a bool variable declared in onnx/utils.py to check whether in onnx export. This variable will be set True while enter torch.onnx.export and set back to false before return. As we do not use torch.onnx.export as it does not support using kwargs in input, we can not modify that bool variable.